HidPkg/UsbHidDxe/UsbHidDxe.c

```diff
@@ -697,6 +697,12 @@ UsbGetFullHidDescriptor (
       }
     }
 
+    // Check if the descriptor length is 0
+    if (DescriptorHeader->Len == 0) {
+      DEBUG ((DEBUG_ERROR, "[%a] Descriptor length is 0\n", __FUNCTION__));

```

Comment:
We're trying to use `__func__` for new changes.

```suggestion
      DEBUG ((DEBUG_ERROR, "[%a] Descriptor length is 0\n", __func__));
```